src/index.html

```diff
@@ -2,21 +2,35 @@
 <html dir="ltr" lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
+    />
     <title>Stencil Component Starter</title>
 
     <!-- modern browsers -->
-    <script async type="module" src="https://js.cdn.manifold.co/@manifoldco/manifold-init/dist/manifold-init/manifold-init.esm.js" ></script>
+    <script
+      nomodule
+      src="https://js.cdn.manifold.co/@manifoldco/manifold-init@0.4.0-beta.1/dist/manifold-init/manifold-init.js"
+    ></script>
```

Comment:
Do the analytics changes prevent this script from loading async? If so we should probably capture that in the README for `manifold-init` and the docs.

Reply:
I haven't tried loading this script async. Analytics doesn't change whether it will work or not, but I don't know whether it worked in the first place. Seems like it should.

Reply:
Yeah that’s a good point. And I guess it’s possible someone could try that in `componentWillLoad` but we don’t need to do that here. Was mostly a “is it possible for _someone_ to break this?” question.